### PR TITLE
improve text formatting

### PR DESCRIPTION
**Pull request by xpepper**

### CODE REVIEW

**Comment by maxim-tschumak** [comment]
@xpepper Are you taking part in the [Hacktoberfest](https://hacktoberfest.digitalocean.com)?

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ export GITHUB_SECRET=your_github_secret
 
 ./build-and-run.sh --bark
 ``` 
-Github webhook endpoint is accessible on http://localhost:8081/github_webhook
+Github webhook endpoint is accessible on http://localhost:8081/github_webhook
```
Comment (ddufourboivin-zalando) [inline README.md]: Why the extra spaces at the end of the line? Especially that this is the only diff of this PR.

**Comment by xpepper** [comment]
> @xpepper Are you taking part in the [Hacktoberfest](https://hacktoberfest.digitalocean.com)?

no, just seeing this minor formatting issue. 
HTH

File: README.md (hunk 1 above)
Comment (ddufourboivin-zalando) [reply README.md]: @xpepper : What is that formatting issue you are mentioning? We only see that you added 4 spaces at the end of a line.

Comment (xpepper) [reply README.md]: @ddufourboivin-zalando yes, those are the 4 spaces that you need to force a new line in github flavored markdown.

Comment (roxspring) [reply README.md]: Ahh - thanks for the explanation!

Seems an unfortunate bit of formatting though - trailing spaces are the kind of things people often "tidy up". Perhaps the two lines are better as separate paragraphs or even bullets.

Comment (xpepper) [reply README.md]: @roxspring yes, indeed! Would you like me to propose a new fix in a new PR?

Comment (ddufourboivin-zalando) [reply README.md]: Then replace that by a fully blank line in between. That is a lot clearer then.

```
a

b
```

would be more readable than

```a    b```.

Comment (xpepper) [reply README.md]: @ddufourboivin-zalando it's fine for me too.

Comment (ddufourboivin-zalando) [reply README.md]: @xpepper : Can you please apply it so we can merge this? It's pending for a while.

**Comment by xpepper** [comment]
I'm going to open a new PR, it'easies

**Pull request closed without merging**